app/submitter/submitter.py

```diff
@@ -141,3 +141,42 @@ def send_message(self, message, tx_id, questionnaire_id, case_id=None):
         finally:
             if connection:
                 self._disconnect(connection)
+
+
+class GCSFeedback:
+    def __init__(self, bucket_name):
+        client = storage.Client()
+        self.bucket = client.get_bucket(bucket_name)
+
+    def upload(self, data, meta_data):
+        blob = self.bucket.blob(meta_data["object_key"])
+        blob.metadata = {
+            "feedback_count": meta_data["feedback_count"],
+            "form_type": meta_data["form_type"],
+            "language_code": meta_data["language_code"],
+            "region_code": meta_data["region_code"],
+            "tx_id": meta_data["tx_id"],
+        }
+        blob.upload_from_string(
+            str(data).encode("utf8"), content_type="application/json"
+        )
+
+        return True
```

Comment:
This should be something like:
```py
    def upload(self, data, metadata):
		logger.info("uploading feedback")

        blob = self.bucket.blob(str(uuid4()))
        blob.metadata = metadata
        blob.upload_from_string(
            str(data).encode("utf8"), content_type="application/json"
        )

        return True

```
The topic around "object_key" is still open for discussion as I don't think it needs to be in the payload. Happy to discuss at tech session.